templates/layout.html

```diff
@@ -41,15 +41,23 @@
     <meta name="google-site-verification" content="58YRXAqZgsPP1fJar_ymC-4xSWtW3O2NmYwuRDzyRpg" />
 </head>
 <body itemscope itemtype="http://schema.org/WebSite">
-    {% if g.MOTD %}
+    {%- if g.MOTD -%}
         <div class="top-bar green pts pbs">
             <div class="w1140p center txtcenter small">
                 <i class="fas fa-bell fa-lg"></i> {{ g.MOTD|safe }}
             </div>
         </div>
-    {% endif %}
+    {%- endif -%}
+
+    {%- if not g.UNDER_MAINTENANCE -%}
+        {%- if g.EVENT -%}
+            <div class="top-bar blue pts pbs">
+                <div class="w1140p center txtcenter small">
+                    <i class="fas fa-calendar-alt fa-lg"></i> {{ g.EVENT.name|safe }} - {{ g.EVENT.datetime.format('MMMM Do, YYYY @ h:mm A ZZZ') }}{% if g.EVENT.server %} on <a href="{{ g.EVENT.server.link }}">{{ macros.server_name(g.EVENT.server, link=False) }}</a>{% endif %}
```

Comment:
If the current datetime is greater or equal than the event datetime -15 minutes, show the server summary